test/gas-analytics/HubPool.RootExecution.ts

```diff
@@ -203,11 +200,68 @@ describe("Gas Analytics: HubPool Root Bundle Execution", function () {
       // Advance time so the request can be executed and execute the request.
       await timer.setCurrentTime(Number(await timer.getCurrentTime()) + consts.refundProposalLiveness + 1);
       const multicallData = leaves.map((leaf) => {
-        return hubPool.interface.encodeFunctionData("executeRootBundle", [leaf, tree.getHexProof(leaf)]);
+        return hubPool.interface.encodeFunctionData("executeRootBundle", [
+          ...Object.values(leaf),
+          tree.getHexProof(leaf),
+        ]);
       });
 
       const receipt = await (await hubPool.connect(dataWorker).multicall(multicallData)).wait();
       console.log(`(average) executeRootBundle-gasUsed: ${receipt.gasUsed.div(REFUND_CHAIN_COUNT)}`);
     });
+    it(`Stress Test: 1 leaf contains ${STRESS_TEST_L1_TOKEN_COUNT} L1 tokens with netSendAmounts > 0`, async function () {
+      // This test should inform the limit # of L1 tokens that we would allow a PoolRebalanceLeaf to contain to avoid
+      // publishing a leaf that is unexecutable due to the block gas limit. Note that this estimate is a bit contrived
+      // and likely an underestimate because we are relaying tokens via the MockAdapter, not an Adapter used for
+      // production.
+
+      // Regarding the block limit, we should target a maximum of 30 million gas. Technically block's can support up
```

Comment:
Similar to the other file, I think this comment is a bit misleading because of the word target.